carmm/run/aims_calculator.py

```diff
@@ -57,7 +60,9 @@ def get_aims_calculator(dimensions, spin=None, relativistic=None, k_grid=None, x
         from ase.calculators.aims import AimsProfile
 
         fhi_calc = Aims(
-            profile=AimsProfile(command=''), # Dummy argument, as command is set to environment variable
+            # Load profile from environment variables
+            profile=AimsProfile(command=os.environ["ASE_AIMS_COMMAND"],
+                                default_species_directory=os.environ["AIMS_SPECIES_DIR"]),
             compute_forces=compute_forces,
```

Comment:
This depends on the OS variables existing. What happens if they haven't been set? Or do we not need to worry about this possibility?

Reply:
I just ran this briefly and it returned the following error:
```
>>> import os
>>> print(os.environ["DUMMY"])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/intel/oneapi/intelpython/latest/lib/python3.9/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'DUMMY'
```

I suggest either some check earlier in the code through a try/except statement checking both environmental variables are set and return an explanatory error.

Reply:
A check for the two environment variables is added.